apps/api/src/app.module.ts

```diff
@@ -19,6 +19,9 @@ import { AuthModule } from "./modules/auth/auth.module";
             playground: true,
             sortSchema: true,
             autoSchemaFile: join(process.cwd(), "schema.graphql"),
+            // https://github.com/nestjs/nest/issues/1905#issuecomment-479431252
+            // @ts-ignore
```

Comment:
Use `@ts-expect-error: <reason>`. It's better for when they eventually (?) fix the issue.

Reply:
ok